### PR TITLE
chore: update webpack and webpack-cli

### PR DESCRIPTION
**Pull request by snitin315**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [x] This is a **test update**
- [ ] This is a **docs update**
- [x] This is a **metadata update**

### For Bugs and Features; did you add new tests?
Yes, updated tests.

### Motivation / Use-Case
update `webpack` and `webpack-cli` to lates version.

### Breaking Changes
None

### Additional Info
No

### CODE REVIEW

File: test/cli/cli.test.js [hunk 1]
```diff
@@ -117,15 +119,6 @@ runCLITest('CLI', () => {
...
   });
 
-  it('unspecified host and port', (done) => {
-    testBin('')
-      .then((output) => {
-        expect(/http:\/\/localhost:[0-9]+/.test(output.stderr)).toEqual(true);
-        done();
-      })
-      .catch(done);
-  });
```
Comment (alexander-akait) [inline test/cli/cli.test.js:127]: We should not delete this tests, let's improve testing here

Comment (snitin315) [reply test/cli/cli.test.js:127]: Now we don't default host so we can change the assertion to false.

Comment (alexander-akait) [reply test/cli/cli.test.js:127]: host should be in IPv6 here

Comment (snitin315) [reply test/cli/cli.test.js:127]: I see `running at http:/` without specifying host and port -

Screenshot at 2021-02-06 10-31-20

Comment (alexander-akait) [reply test/cli/cli.test.js:127]: Bug :smile:

Comment (alexander-akait) [reply test/cli/cli.test.js:127]: We need to fix it, because it is very bad message

Comment (snitin315) [reply test/cli/cli.test.js:127]: Looking into it.

Comment (snitin315) [reply test/cli/cli.test.js:127]: I think we need to use `::` by default here.

Comment (alexander-akait) [reply test/cli/cli.test.js:127]: Ideally we should print message like `Project is running at https://server.com` or `Project is running at IPv4 (IPv6)`, it is in my roadmap, give me today to improve this

Comment (snitin315) [reply test/cli/cli.test.js]: Do we need an update here? 
https://github.com/webpack/webpack-dev-server/blob/b0868c399ee965727a97de303452b9841fd315e3/lib/Server.js#L592-L609

Comment (alexander-akait) [reply test/cli/cli.test.js]: Yes, we need improve it here

Comment (snitin315) [reply test/cli/cli.test.js]:
```js
if (typeof hostname === 'undefined') { 
  prettyHostName = ?? // what should be printed here?
}
```

Comment (alexander-akait) [reply test/cli/cli.test.js]: hm, I need to look at code, why hostname here undefined :confused:

**Review by alexander-akait — APPROVED** [review]
Let's merge as is, to unblock other deps updating and solve messages in the next PR

**Pull request merged**